.github/workflows/rust.yml

```
@@ -23,16 +23,8 @@ jobs:
     - uses: actions/setup-python@v3
     - uses: pre-commit/action@v3.0.1
       with:
-        extra_args: cargo-clippy-fix --all-files
-    - uses: pre-commit/action@v3.0.1
-      with:
-        extra_args: cargo-clippy --all-files
-    - uses: pre-commit/action@v3.0.1
-      with:
-        extra_args: cargo-check --all-files
-    - uses: pre-commit/action@v3.0.1
-      with:
-        extra_args: cargo-fmt --all-files
+        # To be adapted if the structure changes, e.g. new subdirs.
+        extra_args: --files virtme_ng_init/*/*.rs
```

```suggestion
        extra_args: --files virtme_ng_init/**.rs
```

Does this also work? See https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#filter-pattern-cheat-sheet